examples/tests/sdk_tests/container/pull-failure.test.w

```diff
@@ -0,0 +1,8 @@
+bring sim;
+bring cloud;
+
+let x = new sim.Container(
+  name: "pull-failure",
+  image: "foo://do-not-pull-me",
+  containerPort: 1234,
+);
```

Comment:
For any .w files tested by hangar that use containers, add this to the top because we currently can only test them on linux:

```suggestion
/*\
skipPlatforms:
  - win32
  - darwin
\*/

bring sim;
bring cloud;

let x = new sim.Container(
  name: "pull-failure",
  image: "foo://do-not-pull-me",
  containerPort: 1234,
);
```